.gitignore

```diff
@@ -5,4 +5,5 @@
 tests/testthat/results
 .DS_Store
 results/
+data/
```

Comment:
We might also want to add `tests/testthat/data/` here too

Reply:
good call. done.